runner/pool/enterprise.go

```diff
@@ -123,8 +132,16 @@ func (r *enterprise) GetJITConfig(ctx context.Context, instance string, pool par
 		// TODO(gabriel-samfira): Should we make this configurable?
 		WorkFolder: github.String("_work"),
 	}
+	metrics.GithubOperationCount.WithLabelValues(
+		"GenerateEnterpriseJITConfig", // label: operation
+		metricsLabelEnterpriseScope,   // label: scope
+	).Inc()
 	jitConfig, resp, err := r.ghcEnterpriseCli.GenerateEnterpriseJITConfig(ctx, r.cfg.Name, &req)
 	if err != nil {
+		metrics.GithubOperationFailedCount.WithLabelValues(
+			"GenerateEnterpriseJITConfig", // label: operation
+			metricsLabelEnterpriseScope,   // label: scope
+		).Inc()
 		if resp != nil && resp.StatusCode == http.StatusUnauthorized {
 			return nil, nil, fmt.Errorf("failed to get JIT config: %w", err)
```

Comment:
no related to this change.

Note to self: This needs to wrap `runnerErrors.ErrUnauthorized`